scripts/make-subpkg.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Oops I left this as another comment. "Manual. May be rendered useless by opam-publish. TBD."